contentful.json

```diff
@@ -57,23 +57,23 @@
     "Client Side Forms": "No",
     "Plugin System": "No",
     "Customizable UI": "Yes",
-    "Custom Field Types": "No",
+    "Custom Field Types": "Yes",
     "User Management": "Yes",
     "Role Based Permissions": "Yes",
     "Document Level Permissions": "No",
-    "OAuth 2.0 Support": "No",
-    "Auditing": "No",
+    "OAuth 2.0 Support": "Yes",
+    "Auditing": "Yes",
     "API Keys": "Yes",
     "Project Support": "Yes",
     "I18N Localized Content": "Yes",
     "Content Trees": "No",
     "Tagging": "Yes",
-    "Content Relations": "Yes",
-    "Nesting of Fields": "No",
-    "GDPR Compliance": "No",
-    "Editing Conflict Detection": "No",
+    "Content Relations": "1:1, 1:n, n:n, n:1",
```

Comment:
https://www.contentful.com/developers/docs/concepts/links/
https://www.contentful.com/blog/2018/01/16/query-content-graphs-with-incoming-links/
https://www.contentful.com/r/knowledgebase/incoming-links-tagging-content-discovery/

Reply:
I manually merged the `Content Relations` field.